app/models/SuperModel.coffee

```diff
@@ -132,9 +140,13 @@ module.exports = class SuperModel extends Backbone.Model
   # Tracking resources being loaded for this supermodel
 
   finished: ->
-    return @progress is 1.0 or not @denom
+    return (@progress is 1.0) or (not @denom) or @failed 
```

Comment:
Nah, just something to keep in mind might be the cause of bugs if we see anything weird like that.